### PR TITLE
Create briggsy1213.json

### LINKED ISSUE #1594: Adding Ontario
## Adding Toronto to the map

Hi everyone! This issue is to track the addition of Toronto to the map, which can be found at:

- Longitude: 43.652017
- Latitude: -79.372639

### PR DESCRIPTION
**Pull request by Briggsy1213**
closes #1594

### CODE REVIEW

**Comment by willmerrr** [comment]
Well done mate! Love it! Keep it up! Proud of you! :sparkling_heart: 
وحدة ، حرية ، اشتراكية

**Comment by Olkrm** [comment]
:-1: terrible tbh

**Comment by loc3219** [comment]
YOU GOD, BEAST AT CODING :space_invader: :100:

File: _pins/briggsy1213.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: briggsy1213
+latitude: 43.652017
+longitude: -79.37263943.652017
```
Comment (crichID) [inline _pins/briggsy1213.json]: @Briggsy1213 it looks like you have some extra digits here (with an extra decimal point). You will need to remove these in order for the pin to work properly. Please fix it and then mention me to approve the changes :wink:

**Comment by shyam-ramakrishnan** [comment]
Cool! :)

**Pull request merged**